cpp/src/codegen/CMakeLists.txt

```diff
@@ -55,6 +55,7 @@ target_include_directories(gandiva_obj_lib
     ${CMAKE_SOURCE_DIR}/src
     $<TARGET_PROPERTY:LLVM::LLVM_INTERFACE,INTERFACE_INCLUDE_DIRECTORIES>
     $<TARGET_PROPERTY:ARROW::ARROW_SHARED,INTERFACE_INCLUDE_DIRECTORIES>
+    ${CMAKE_BINARY_DIR}/googletest-src/googletest/include
```

Comment:
for uniformity, can you please use the same syntax as the dependencies ?

$<TARGET_PROPERTY:gtest_main,INCLUDE_DIRECTORIES>

Reply:
Thanks @pravindra for checking.
I have fixed it and also confirmed compilation passed.